### PR TITLE
feat: add pull request template 

### LINKED ISSUE #7: Add a pull request template
This would aid contributors on submitting their PR, especially if it’s their first time contributing to an open source project.

### PR DESCRIPTION
**Pull request by CBID2**
## Issue
Closes #7

## Proposed Changes 
This PR creates a pull request template, making it easy for contributors to present their contributions effectively.

## Notes to Reviewer

N/A

### CODE REVIEW

**Comment by ykdojo** [comment]
Will close this one soon if there's no update.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Issue
+<!-- Use `Closes #issue number` if your PR fixes an issue or `Relates to #issue number` if it connects to the issue number -->
+
+## Changes proposed
+<!-- List all the proposed changes in your PR -->
+
+## Screenshots
+<!-- Add all the screenshots that support your PR -->
+
+## Note to Reviewers
```
[the remark this answers is not in the document]
Comment (Devanshukoli) [reply .github/pull_request_template.md:10]: > Let's keep the casing consistent and use sentence casing here

https://github.com/ykdojo/OpenStream/issues/7#issuecomment-1929111470

* here, is this what you are referring to?

**Comment by CBID2** [comment]
> Will close this one soon if there's no update.

Sorry @ykdojo. Made the update

**Comment by Devanshukoli** [comment]
oh, Just show the *fix* part, got it what it was. :)

**Pull request merged**